src/types/commons.ts

```diff
@@ -12,3 +12,7 @@ export type SuccessResponse = t.TypeOf<typeof SuccessResponse>;
 
 export const STRINGS_RECORD = t.record(t.string, t.string);
 export type STRINGS_RECORD = t.TypeOf<typeof STRINGS_RECORD>;
+
+// Type from passport Authenticator used as response of session and authenticate methods
+// tslint:disable-next-line: no-any
+export type AuthenticateRet = any;
```

Comment:
why this alias ?

Reply:
`AuthenticateRet` is not exported into `@types/passport` and his value is `any`. We can use a pure `any` instead to create an alias. What do you suggest @gunzip ?

Reply:
that's ok to use a pure any[](url)

Reply:
Fixed in 8019d63